modules/installing-ove-console.adoc

```diff
@@ -0,0 +1,104 @@
+
+.Prerequisites
+
+* You have the URL of the installation web console that was provided by the rendezvous node.
+
+.Procedure
+
+. In a web browser, go to the URL provided by the rendezvous node.
+
+. Configure your cluster in the *Cluster details* page:
+
+.. Enter a name for the cluster in the *Cluster name* field.
+
+.. Enter a base domain for the cluster in the *Base domain* field.
+All subdomains for the cluster will use this base domain.
++
+[NOTE]
+====
+The base domain must be a valid DNS name.
+You must not have a wildcard domain set up for the base domain.
+====
+
+.. Enter your pull secret in the *Pull secret* field.
+You can obtain a copy of your pull secret link:https://console.redhat.com/openshift/install/pull-secret[here].
+
+.. Optional: In the *Number of control plane nodes* field, select the number of control plane nodes for your installation from the dropdown menu.
+The default value is `3`.
+
+.. Optional: Select the *include custom manifests* checkbox if you want to upload custom manifests to further configure your cluster.
+This option adds an additional page for custom manifests that you use later in the configuration process.
++
+[IMPORTANT]
+====
+If you have already added custom manifests, clearing the *Include custom manifests* checkbox automatically deletes them all.
+You must confirm the deletion.
+====
+
+.. Click *Next* to continue.
+
+. Choose additional Operators to install in the *Operators* page.
+
+.. If you want to install all of the Operators recommended for running Virtualization on your cluster, select *Virtualization* in the *Bundles* section.
+
+.. If you want to install only some Operators, select the individual Operators from the Single Operators section.
++
+[NOTE]
+====
+Some of the listed Operators are available only as part of the Virtualization Operator bundle.
+====
+
+.. Click *Next* to continue.
+
+. Configure the hosts in the *Host discovery* page:
+
+.. Verify that every machine you booted from the ISO appears in the *Host Inventory* section and has a *Status* value of *Ready*.
+
+.. For each host, click the expand icon and verify that all of the specification fields are correct.
+
+.. Optional: For each host configure the role selecting an option from the dropdown menu of the *Role* column.
+The default value for every host except the rendezvous node is `Auto-assign`.
+
+.. Click *Next* to continue.
+
+. On the *Storage* page, click the expand icon for each host and verify that all of the specification fields are correct.
+
+. Click *Next* to continue.
+
+. Configure the cluster networking on the *Networking* page:
+
+.. Optional: Select a machine network from the dropdown menu of the *Machine network* field.
+Otherwise, a default value is selected.
+
+.. Enter an IP address in the *API IP* field.
+An API IP provides an endpoint for all users to interact with, and configure the platform.
```

Comment:
Should probably be either:
An API IP provides an endpoint for all users to interact with, and configure, the platform.
Or 
An API IP provides an endpoint for all users to interact with and configure the platform.

Reply:
Good catch, thanks! Just an FYI that this line was pulled from the [assisted docs](https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2025/html/installing_openshift_container_platform_with_the_assisted_installer/installing-with-ui#configuring-networking_installing-with-ui:~:text=An%20API%20virtual%20IP%20provides%20an%20endpoint%20for%20all%20users%20to%20interact%20with%2C%20and%20configure%20the%20platform.), so that grammar error also exists there

Reply:
Thanks for letting us know. We'll change it.